### ISSUE TITLE
Update gdm-prime to gnome 46

### ISSUE THREAD

**Issue by cucumber-sp** [Askannz/optimus-manager#559]
Gdm prime is broken for new gnome version, please update it

**Comment by truly-irham** [comment]
Hi, I have pushed the updated PKGBUILD to update gdm-prime to version 46.0, which is compatible with gnome 46 (I have tested this myself and the updated gdm-prime works in gnome 46). Feel free to take a look at my repo: https://github.com/truly-irham/gdm-prime and do `makepkg -si`, or simply download and install the pkg.tar.zst files from my releases: https://github.com/truly-irham/gdm-prime/releases/tag/v46.0. I hope my small contribution helps while waiting for Askannz to release the official updated aur package. Cheers!
